frontend/src/app.jsx

```diff
@@ -62,7 +48,7 @@ export default function App() {
       </form>
       <div className="task">
         {tasks.map((task) => (
-            <Task key={task._id} task={task} setTasks={setTasks} update={update}  setUpdate={setUpdate} />
+            <Task key={task._id} task={task} setTasks={setTasks} />
```

Comment:
We're no longer using the `update` and `setUpdate` state variables as explained [in this comment](https://github.com/nds-fsd/Abd-ha-todo-app/pull/3/files#r1505501386).